scripts/install_moodle.sh

```diff
@@ -861,6 +861,8 @@ EOF
 
    # setting file locking to false if fileServerType is set to nfs
    if [ $fileServerType = "nfs" ]; then
+      sed -i "23 a \$CFG->preventfilelocking = true;" /moodle/html/moodle/config.php
```

Comment:
Then, are you now acknowledging that the original change was incorrect? Not only that, if NFS does support flock() with newer versions of Linux kernels (>=2.6.12, which is be virtually all current Linux kernels), should we really set this true? As the `config-dist.php` says, file locking is still important for cron jobs and shouldn't be prevented when the underlying persistent store (e.g., NFS on newer Linux kernels) supports it well, so I don't think we should set this true at all for our use case.

Please do show us a proof that the installed Moodle deployment will be problematic without this on NFS. We ourselves have never had any file locking issues with NFS-based deployments, and I myself still don't understand why this is needed.